### DISCUSSION TITLE
Digital Ocean Credit

### DISCUSSION THREAD

**Discussion by caraka1928** [community/community#40699] [GitHub Education]
i have linked my github with my digitalocean. why is my credit not shown in my DO account

**Comment by ossanna16** [comment]
Hi there @caraka1928 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
